broker/src/main/java/org/apache/rocketmq/broker/processor/PullMessageProcessor.java

```diff
@@ -87,8 +87,8 @@ private RemotingCommand processRequest(final Channel channel, RemotingCommand re
 
         response.setOpaque(request.getOpaque());
 
-        if (LOG.isDebugEnabled()) {
-            LOG.debug("receive PullMessage request command, {}", request);
+        if (log.isDebugEnabled()) {
```

Comment:
For this particular place, this can be removed, I agree. But it is good to concenrate the attention on what PR is about.